website_portal_medical_patient/security/ir.model.access.csv

```diff
@@ -1,4 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_medical_patient_portal,medical.patient.portal,medical.model_medical_patient,base.group_portal,1,1,1,0
-access_res_partner_portal,res.partner.portal,base.model_res_partner,base.group_portal,1,1,1,0
-access_ir_sequence_portal,ir.sequence.portal,base.model_ir_sequence,base.group_portal,1,1,0,0
+access_res_partner_portal,res.partner.portal,base.model_res_partner,base.group_portal,1,1,1,0
```

Comment:
This was saved directly from Excel, so I argue against this because it's not code.